python/ray/dataframe/dataframe.py

```diff
@@ -776,9 +776,18 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
             DataFrame with the dropna applied.
         """
         if is_list_like(axis):
-            raise NotImplementedError(
-                "To contribute to Pandas on Ray, please visit "
-                "github.com/ray-project/ray.")
+            result = self
+            for ax in axis:  # TODO: inefficient, df built as intermediate
```

Comment:
prefer comment: `# TODO(kunalgosar): this builds an intermediate dataframe, which does unnecessary computation`

Also, put comment on its own line.